cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -902,6 +909,34 @@ protected Future<Void> clusterId(KafkaStatus kafkaStatus) {
                 });
     }
 
+    /**
+     * Configures the default users quota in Kafka in case that the {@link QuotasPluginKafka} is used
+     *
+     * @return  Future which completes when the default quotas are configured
+     */
+    protected Future<Void> defaultQuotas() {
+        QuotasPlugin quotas = kafka.quotas();
+
+        if (quotas instanceof QuotasPluginKafka quotasPluginKafka) {
```

Comment:
We can also use getType() here, instead of instanceof.

Reply:
same as above